scripts/parks/ParkList.js

```diff
@@ -1,24 +1,33 @@
-// imports useParks() and getParks() and ParkHTML()
 import { getPark, usePark } from "./ParkProvider.js";
-// generates HTML list of all individual ParkHTML() components
-
-// exports ParksList() which renders the park list to the DOM
-
-// listens for browser generated "click" event at target with class name "button"
-
-//
-
-
-
-eventHub.addEventListener("click", (btnClickedEvent) => {
-
-      if(btnClickedEvent.target.classList.contains("btn--nat-park")) {
-
-            const parkDeetsClickEvent = new CustomEvent("parkDeetsBtnClicked", {
-            })
-            eventHub.dispatchEvent(parkDeetsClickEvent)
-      }
-      else {
-            return false
-      };
-})
\ No newline at end of file
+import { ParkHTML } from "./ParkHTMLGenerator.js";
+
+const contentTarget = document.querySelector(".container--nat-park")
```

Comment:
Pulled branch and confirmed the code files are sound. Served to browser and verified the park drop down list populates with the appropriate park data. Verified that the itinerary preview section is populated with the park data upon selection.